net.jami.Jami.json

```diff
@@ -0,0 +1,179 @@
+{
+	"app-id": "net.jami.Jami",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "3.28",
```

Comment:
Updte runtime to 3.32

Reply:
I cannot at the moment, as the application crashes when I do that and I still wasn't able to download `org.gnome.Sdk.Debug` to debug this (503 Service Unavailable). See also my last [commit](https://gitlab.com/ntninja/net.jami.jami/commit/63a611df12a8eddbb4acc9d562a85e2c79faeb8d) regarding this and discussions on possibly related issues: https://github.com/flathub/cx.ring.Ring/issues/2 & https://github.com/flathub/cx.ring.Ring/issues/3.